winpr/libwinpr/synch/wait.c

```diff
@@ -193,8 +217,10 @@ DWORD WaitForSingleObject(HANDLE hHandle, DWORD dwMilliseconds)
 	{
 		WINPR_PROCESS *process;
 		process = (WINPR_PROCESS *) Object;
+		int rc;
```

Comment:
is the int rc here really required ?
Would not be (waitpid(process->pid, &(process->status), 0) != process->pid enought ?